recipes/finam/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "finam" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/finam-{{ version }}.tar.gz
+  sha256: 6adc4b3fb0bafd7115be21cc3fd90a14295e4b9602a4ff0e1627be8210924533
+  patches:
+    - skip_c_log_test_win.patch  # [win]
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - hatchling >=1.8.0
+    - hatch-vcs
+    - pip
+  run:
+    - python >=3.8
+    - numpy >=1.14.5
+    - scipy >=1.9.0
+    - pyevtk >=1.1.1
+    - pint >=0.20.0
+    - pyproj >=3.4
+    - pandas >=1.3
+    - opensimplex >=0.4
+
+test:
+  imports:
+    - finam
+  requires:
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pytest -vv tests
+
+about:
+  summary: FINAM is not a model.
+  home: https://finam.pages.ufz.de
+  doc_url: https://finam.pages.ufz.de/finam
+  dev_url: https://git.ufz.de/FINAM/finam
+  description: |
+    FINAM is an open-source component-based model coupling framework for environmental models.
+    It aims at enabling bi-directional online couplings of models for different compartments like
+    geo-, hydro-, pedo- and biosphere.
+  license: LGPL-3.0-or-later
+  license_family: LGPL
+  license_file:
+    - COPYING
+    - COPYING.LESSER
+    - LICENSE.md
+    - AUTHORS.md
```

Comment:
Is this a license file?

Reply:
These are all the files detected by most built-tools as default license files. So, yes.

Reply:
What tools? Looks like a false positive: https://git.ufz.de/FINAM/finam/-/blob/v0.4.0/AUTHORS.md

Please remove this file from the license file list.

Reply:
PEP639 specifies this: https://peps.python.org/pep-0639/#add-license-files-key
```
license-files.globs = ["LICEN[CS]E*", "COPYING*", "NOTICE*", "AUTHORS*"]
```
At some point the authors should be mentioned, right?
Tools that respect this: [hatch](https://hatch.pypa.io/latest/config/metadata/#license), [scikit-build-core](https://scikit-build-core.readthedocs.io/en/latest/configuration.html#customizing-the-built-wheel), [setuptools](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#setuptools-specific-configuration)

Reply:
Sorry, I'm not sure why a PEP is relevant for the conda-forge package. They authors can, and should be mentioned, but they are not a license information!

Reply:
OK, then I will remove the `AUTHORS.md` from that list.

Reply:
Feel free to include that file in the code or package metadata like others do. It is also in the source code anyways, easy to find. I'm sorry but that PEP is for PyPI and, IMO, useless. Just add noise and extra files, making parsing for licenses very difficult for no reason.

Reply:
PS: I read the pep quickly and it seems that it accommodates the fact that some folks add that file in the license list, not a recommended metadata. It seems more like a catch-all approach so folks won't break existing workflows.

Reply:
Removed.